src/webviews/apps/plus/graph/graph.scss

```diff
@@ -7,3 +7,8 @@
 @import '../../shared/codicons';
 @import '../../shared/utils';
 @import '../../../../../node_modules/@axosoft/gitkraken-components/lib/css/GKGraphStyles.css';
+
+// override important font size from base.scss
+#graph-container-helmet {
+    font-size: 14px;
```

Comment:
I think this is not the correct way to fix that. The fix should be on the library side.

And then, the css will be included on our generated `GKGraphStyles.css` file.

Reply:
My thought was that CSS specific to the gitlens app was messing with our CSS styling, so adding the CSS rule inside of  gitlens seemed fitting (this rule won't necessarily be needed when we use the graph in other places). But, I'm willing to move this CSS to the component.